andela/selene-ah-backend
# Document all models on swagger

#### What does this PR do?
It documents all the project models on swagger ui

#### Description of Task to be completed?
- [x] Document all models
- [x] Set the paths

#### How should this be manually tested?
start the server ```npm start```
visit ```http://localhost:3000/docs```
All the models should be visible

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/23301584/50684843-476b0900-1017-11e9-91dc-2cfffee4e35a.png)
